Neo4j.Driver/Neo4j.Driver.Tests/ConnectionPoolTests.cs

```diff
@@ -665,8 +665,8 @@ public class NumberOfInUseConnections
             public void ShouldReturnZeroAfterCreation()
             {
                 var uri = new Uri("localhost:7687");
-                var connectionSettings = new ConnectionSettings(uri, AuthTokens.None, Config.DefaultConfig);
-                var poolSettings = new ConnectionPoolSettings(1, 1, TimeSpan.MaxValue, TimeSpan.MaxValue);
+                var connectionSettings = new ConnectionSettings(AuthTokens.None, Config.DefaultConfig);
+                var poolSettings = new ConnectionPoolSettings(1, 1, Config.Infinite, Config.Infinite, Config.Infinite);
```

Comment:
We may elaborate on what TimeSpan value makes it Infinite. I see the point that we accept -1 as an Infinite, but since we define the type of setting timeout values as TimeSpans, any TimeSpan value less than zero, zero itself or even TimeSpan.MaxValue could be perceived as infinite values. Not a big overhead when set something different than -1, but we shall be more natural.